CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+
+### Pull Requests
+
+1. **Fork the repository** and create your branch from `master`.
+
+2. **Make your changes**: Apply your changes, following the coding conventions described below.
+
+3. **Commit your changes**: Commit your changes using a descriptive commit message.
+
+4. **Open a Pull Request**: Describe what you did in the pull request description. Mention the issue number if your pull request is related to an existing issue.
+
+5. **Include Screenshots**: If your pull request includes any visual changes to the project, please include before and after screenshots in your pull request description to help us better understand the changes.
+
+6. **Wait for review**: Once your pull request is opened, it will be reviewed as soon as possible. Changes may be requested, and your responsiveness is appreciated.
+
+## Setting up your development environment
+
+### Installing dependencies
+
+First, ensure that the following are installed globally on your machine:
+
+- [Node.js 20.11+](https://nodejs.org/en/download/releases)
+- [Yarn v3](https://yarnpkg.com/getting-started/install)
+
+Then, from the root folder run:
+
+- `yarn install` to install dependencies
+- `yarn build` to build the package
+- `yarn dev` to start the example app and build the package with hot reloading
+- `yarn test` to ensure that the test suite runs correctly
+
+### Navigating the codebase
+
+The SDK is built using yarn workspaces. 
+
+- `packages/wallet-sdk` - The main package that exports the SDK
+- `examples/test-app` - An example React app that is used to test the SDK in a real browser environment
+
+Use `yarn dev` to start the example app and build the package with hot reloading.
+
+### Linting and formatting
+
+This project uses [Biome](https://github.com/biomejs/biome) for linting and formatting. See the [Biome docs](https://biomejs.dev/guides/editors/first-party-extensions/) for more information on how to configure your editor to use Biome.
+
+### Signing Commits
+
+All commits need to be signed with a GPG key. This adds a second factor of authentication that proves that it came from
+you, and not someone who managed to compromise your GitHub account. You can enable signing by following these steps:
+
+1. [Generate GPG Keys](https://help.github.com/en/github/authenticating-to-github/generating-a-new-gpg-key) and upload them to your Github account, [GPG Suite](https://gpgtools.org/) is recommended for OSX
+
+2. Use `gpg-agent` to remember your password locally
+
+```bash
+vi ~/.gnupg/gpg-agent.conf
+
+default-cache-ttl 100000000
+max-cache-ttl 100000000
+```
+
+3. Configure Git to [use your keys when signing](https://help.github.com/en/github/authenticating-to-github/telling-git-about-your-signing-key).
+
+4. Configure Git to always sign commits by running `git config --global commit.gpgsign true`
+
+5. Commit all changes with your usual git commands and you should see a `Verified` badge near your commits
```

Comment:
I think we can simplify this section steps to just linking this guide https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification

Reply:
Good callout +1